boolean_algebra/array_complement.py

```diff
@@ -0,0 +1,23 @@
+def array_complement(initial_lst: list, diff_lst: list) -> list:
+    """
+    removes all values from list initial_lst which
+    are also present in list diff_lst [ while keeping
+    the initial order of the elements ]
+
+    https://en.wikipedia.org/wiki/Complement_(set_theory)
+
+    >>> array_complement([1, 6, 2, 8], [1, 4])
+    [6, 2, 8]
+    >>> array_complement([146], [146, 25])
+    []
+    >>> array_complement([35], [36])
+    [35]
+
+
+    """
+
+    return [c for c in initial_lst if c not in diff_lst]
```

Comment:
This doesn't look like `boolean_algebra`. This could be in `array`? if there's any update possible to existing code

Reply:
thank you for the suggestion! sounds like a nice idea! would it be okay if i create a new "array" folder in the repo for this? or did you mean that there might be a smilar folder, which exists already, and i should add this algorithm to it? if so, could you please direct me to it?